app/lib/differential_expression_service.rb

```diff
@@ -1,5 +1,116 @@
 # handle launching differential expression ingest jobs
 class DifferentialExpressionService
+  # run a differential expression job for a given study on the default cluster/annotation
+  #
+  # * *params*
+  #   - +study_accession+ (String) => Accession of study to use
+  #   - +user+ (User) => Corresponding user, will default to study owner
+  #
+  # * *yields*
+  #   - (IngestJob) => Differential expression job in PAPI
+  #
+  # * *returns*
+  #   - (Boolean) => True if job queues successfully
+  #
+  # * *raises*
+  #   - (ArgumentError) => if requested parameters do not validate
+  def self.run_differential_expression_on_default(study_accession, user: nil)
+    study = Study.find_by(accession: study_accession)
+    validate_study(study)
+    raise ArgumentError, "#{study.accession} has no default cluster" if study.default_cluster.blank?
+    raise ArgumentError, "#{study.accession} has no default annotation" if study.default_annotation.blank?
+
+    annotation_name, annotation_type, annotation_scope = study.default_annotation.split('--')
+    annotation = {
+      annotation_name: annotation_name,
+      annotation_type: annotation_type,
```

Comment:
Any objections to removing this parameter from the DE pipeline, @jlchang?  It'd reduce cognitive load for maintainers.  Context [here](https://github.com/broadinstitute/single_cell_portal_core/pull/1483#discussion_r868173916).

Reply:
I have no arguments, but that would require upstream edits in `scp-ingest-pipeline`, as you noted.  I think that can be done later and doesn't necessarily need to be done right now.